src/main/java/cz/brmlab/yodaqa/flow/dashboard/SourceIDGenerator.java

```diff
@@ -0,0 +1,23 @@
+package cz.brmlab.yodaqa.flow.dashboard;
+
+/**
+ * Created by nhl on 7/9/15.
+ */
```

Comment:
I think we break things when we have two class-level javadocs. Let's drop the first one?